public/app/features/dashboard-scene/serialization/transformSaveModelSchemaV2ToScene.ts

```diff
@@ -180,7 +180,7 @@ export function transformSaveModelSchemaV2ToScene(dto: DashboardWithAccessInfo<D
         preserveDashboardSceneStateInLocalStorage,
         addPanelsOnLoadBehavior,
         new DashboardReloadBehavior({
-          reloadOnParamsChange: config.featureToggles.reloadDashboardsOnParamsChange,
+          reloadOnParamsChange: config.featureToggles.reloadDashboardsOnParamsChange && false,
```

Comment:
@ivanortegaalba FYI i pushed this. v2 does not currently have reload implemented and this caused runtime errors in the console. I've opened https://github.com/grafana/grafana/issues/103519 to followup

Reply:
@bfmatei you implemented v1 reload, do you want to follow up on this one?